app/components/common/Feedback.tsx

```diff
@@ -382,12 +341,12 @@ const FeedbackButton = () => {
           indicatorColor={smColors.red}
         >
           <ModalContainer>
-            <InputWrapper label="Name" required>
+            <InputWrapper label="Discord handle" required>
```

Comment:
I object here. Not everyone has a Discord account, we have thousands of users, we shouldn’t miss the reports from those who will think that they cannot report an issue because they can’t fill one field or they are excluded or smth. Either it shouldn’t be required or it should not ask for Discord handle only - maybe 
```suggestion
            <InputWrapper label="Your name or Discord handle" required>
```